civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -562,14 +582,49 @@ function civihr_employee_portal_init() {
       '/lib/sweetalert/sweetalert.min.js'
     ]);
 
+    _add_module_script_to_page([
```

Comment:
you should reuse the already existing `_add_script_to_page`. Just change the signature
```php
function _add_script_to_page($resources, $extension = NULL, $page = NULL) {

}

// ...

_add_script_to_page('reqangular.min.js', 'org.civicrm.reqangular', 'dashboard');
_add_script_to_page(['/js/tasks.js', '/js/ta-documents-app.js'], NULL, 'dashboard');
```
might not be the most elegant solution, but it's better define a whole new function